mlir/lib/Dialect/MIOpen/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -84,6 +84,21 @@ void computeLoadStoreTypeInfo(OpBuilder &b, ArrayRef<int64_t> sliceLengths,
   loadType = vectorTypeOrSelf(elementType, loadLength);
 }
 
+Type obtainAccumulatorType(OpBuilder &b, Type &elementType, Type &destType) {
```

Comment:
In minor nitpicks, we could put a `static` on this.

Reply:
Good idea. I debated over this as I see none of other local function carry static. While I'm here, better fix those as well.

Fixed in ac59dec